packages/side-nav/src/vaadin-side-nav-item.d.ts

```diff
@@ -110,6 +110,16 @@ declare class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixi
    */
   target: string | null | undefined;
 
+  /**
+   * Whether to exclude the item from client-side routing. When enabled,
+   * this causes the item to behave like a regular anchor, causing a full
+   * page reload. This only works with supported routers, such as the one
+   * provided in Vaadin apps, or when using the side nav `onNavigate` hook.
+   *
+   * @type {boolean}
+   */
+  routerIgnore: boolean;
```

Comment:
Let's add the `@attr` JSDoc annotation for VSCode Lit plugin to indicate dash-cased attribute:

```
   * @attr {boolean} router-ignore
```